src/components/button/Button.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Button from './index';
+
+it('renders correctly', () => {
+  const tree = renderer.create(<Button text={' '} />).toJSON();
```

Comment:
it showed me `console.error node_modules/prop-types/checkPropTypes.js` so i do this to pass it ,should we download enzyme ? @MuhannedNoman